app/detail/Pane.tsx

```diff
@@ -0,0 +1,74 @@
+import {BrimEvent} from "ppl/detail/models/BrimEvent"
+import {Correlation} from "ppl/detail/models/Correlation"
+import {SuricataEvent} from "ppl/detail/models/SuricataEvent"
+import {ZeekEvent} from "ppl/detail/models/ZeekEvent"
+import RelatedAlerts from "ppl/detail/RelatedAlerts"
+import RelatedConns from "ppl/detail/RelatedConns"
+import CorrelationPanel from "ppl/detail/Correlation"
+import React, {useLayoutEffect, memo, useMemo, useRef} from "react"
+import {useSelector} from "react-redux"
+import ConnPanel from "src/js/components/LogDetails/ConnPanel"
+import {Md5Panel} from "src/js/components/LogDetails/Md5Panel"
+import LogDetails from "src/js/state/LogDetails"
+import styled from "styled-components"
+import {zng} from "zealot"
+import Fields from "./Fields"
+import NoSelection from "./NoSelection"
+
+const BG = styled.div`
+  padding: 12px;
+
+  section {
+    margin-bottom: 24px;
+  }
+`
+
+type Props = {
+  record: zng.Record
+}
+
+const Content = memo<Props>(function Content({record}) {
+  const event = useMemo(() => BrimEvent.build(record), [record])
+  const isZeek = event instanceof ZeekEvent
+  const isSuricata = event instanceof SuricataEvent
+  const {uid, cid} = new Correlation(record).getIds()
+  const isConn = isZeek && record.try("_path").toString() === "conn"
+  const hasMd5 = isZeek && record.has("md5")
+
+  return (
+    <BG className="detail-pane-content">
+      <div className="column">
+        <Fields record={record} />
+      </div>
+      <div className="column">
```

Comment:
These column classes do nothing in the search window, but get styled in the detail window to provide that two column look.

<img width="812" alt="image" src="https://user-images.githubusercontent.com/3460638/103387893-18c78200-4abb-11eb-8646-250e40fd8c9a.png">